visual-studio-dev-vm-O365/scripts/Sample.ps1

```diff
@@ -0,0 +1,28 @@
+﻿cls
```

Comment:
remove this file from the PR and ensure the template can be deployed with the script in the root of the repo (ps1 & sh)